File: JUnit_Tuts/junitdemo/src/main/java/com/Calculator.java
package com;

public class Calculator {
	Calculator(){
		System.out.println("<<---------------Calculator Object Created-------------->>");
	}
	public int add(int a, int b) {
		return a + b;
	}
	public int sub(int a, int b) {
		return a - b;
	}
	public int mul(int a, int b) {
		return a * b;
	}
	public int div(int a, int b) throws ArithmeticException{
		if (b!= 0)
			return a / b;
		else
			throw new ArithmeticException();
	}
	public int mod(int a, int b) {
		return a % b;
	}
}

File: AngularTuts/test-demo/src/app/calculator.ts
export class Calculator{
    add (a: number, b: number): number {
        return a + b;
    }
    sub (a: number, b: number): number {
        return a - b;
    }
    mul (a: number, b: number): number {
        return a * b;
    }
    div (a: number, b: number): number {
        if (b == 0) {
            return 0;
        }
        return a / b;
    }
}